pkg/policies/opa/rego/docker/docker_add/CurlWgetinsteadofAdd.rego

```diff
@@ -0,0 +1,8 @@
+package accurics
+
+{{.prefix}}{{.name}}{{.suffix}}[run.id]{
+	run := input.add[_]
+    config := run.config
+    
+    re_match("https?", config)
```

Comment:
value of `config` variable may or may not be in form of an array, need to create checks for both possibilities. please keep edge cases in mind while writing policy code